content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,92 @@
+- **Added** a Grafana dashboard for Citadel monitoring.
+- **Added** [metrics](/reference/commands/sidecar-injector/#metrics) for monitoring the sidecar injector webhook.
+- **Added** control plane metrics to monitor Istio's configuration state.
+- **Added** telemetry reporting for traffic destined to the `Passthrough` and `BlackHole` clusters.
+- **Added** alpha support for in-proxy generation of service metrics using Prometheus.
+- **Added** alpha support for environmental metadata in Envoy node metadata.
+- **Added** alpha support for Proxy Metadata Exchange.
+- **Added** alpha support for the OpenCensus trace driver.
+- **Improved** the mesh dashboard to provide monitoring of Istio's configuration state.
+- **Improved** the Pilot dashboard to expose additional key metrics to more clearly identify errors.
+- **Removed** deprecated `Adapter` and `Template` custom resource definitions (CRDs).
+- **Deprecated** Mixer adapters. Please begin the transition of all extensions to the new extension model. Legacy Mixer integration support continues only for Istio 1.3 and 1.4.
+- **Deprecated** Deprecated HTTP API Spec. The support for producing API attributes will be be removed in Istio 1.4.
+
+## Policy
+
+- **Improved** rate limit enforcement to allow communication when the quota backend is unavailable.
+
+## Configuration Management
+
+- **Added** a configuration for Galley to use new configuration processing.
+- **Fixed** Galley to stop too many gRPC pings from closing connections.
+- **Improved** Galley to avoid control plane upgrade failures.
+
+## istioctl
+
+- **Added** `istioctl kube-uninject` to remove Istio configuration from a previously injected configuration.
+- **Added** `istioctl experimental describe pod` to produce the Istio configuration affecting a Pod.
+- **Added** `istioctl experimental manifest apply` to generate and apply an Istio installation.
+- **Added** `istioctl experimental manifest diff` to show the difference betweeen two manifests.
+- **Added** `istioctl experimental manifest generate` to generate an Istio installation manifest.
+- **Added** `istioctl experimental manifest migrate` to convert Istio's `values.yaml` file into a format that suits the Istio operator.
+- **Added** `istioctl experimental manifest versions` to list all supported Istio versions.
+- **Added** `istioctl experimental profile diff Show` to show the difference between two Istio configuration profiles.
+- **Added** `istioctl experimental profile dump` to show an Istio configuration profile.
+- **Added** `istioctl experimental profile list` to list all Istio configuration profiles.
+- **Added** `istioctl experimental add-to-mesh` to add virtual machines to an existing Istio service mesh.
+- **Added** `istioctl experimental remove-from-mesh` to remove virtual machines from an existing Istio service mesh.
+- **Promoted** the `istioctl` subcommands `metrics`, `convert-ingress`, and `dashboard` to supported features.
+
+## Documentation
```

Comment:
We haven't traditionally documented doc changes. I don't think this is worth it, and it's quite incomplete anyway. So I propose to remove this section.

Reply:
I'll remove for now, if there is discord about the docs, we can address in slack.